app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt

```diff
@@ -553,19 +558,23 @@ class ComposeActivity :
         super.onSaveInstanceState(outState)
     }
 
-    private fun displayTransientError(@StringRes stringId: Int) {
-        val bar = Snackbar.make(binding.activityCompose, stringId, Snackbar.LENGTH_LONG)
+    private fun displayTransientError(errorMessage: String) {
+        val bar = Snackbar.make(binding.activityCompose, errorMessage, Snackbar.LENGTH_LONG)
         // necessary so snackbar is shown over everything
         bar.view.elevation = resources.getDimension(R.dimen.compose_activity_snackbar_elevation)
+        bar.setAnchorView(R.id.composeBottomBar)
```

Comment:
this will prevent overlapping now, right?

Reply:
yes